tsconfig.json

```diff
@@ -1,8 +1,20 @@
 {
-	"extends": "@hashicorp/platform-configs/tsconfig/next.json",
 	"compilerOptions": {
 		"baseUrl": "src",
-		"strict": false
+		"strict": false,
+		"target": "es5",
+		"incremental": true,
+		"module": "esnext",
+		"lib": ["dom", "dom.iterable", "esnext"],
+		"jsx": "preserve",
+		"allowJs": true,
+		"esModuleInterop": true,
+		"forceConsistentCasingInFileNames": true,
+		"isolatedModules": true,
+		"moduleResolution": "node",
+		"resolveJsonModule": true,
+		"skipLibCheck": true,
+		"noEmit": true
```

Comment:
nice to be explicit here on our setting versus just taking them from another project.

Reply:
@dstaley mind going all the way and moving over the platform configs for Jest too from `@hashicorp/platform-configs`? That way we can fully remove the dependency `@hashicorp/platform-configs`. (No is an okay answer here, we can do that work in another PR.)

Reply:
Let's save that for a future PR as I'd like to evaluate the need for that package across all of our codebases and come up with an approach that works for both `dev-portal` and `web`!